Index: lib/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/CMakeLists.txt	(revision 70ca710c1111a48f03befe184b873022a9c048e2)
+++ lib/CMakeLists.txt	(date 1570482208551)
@@ -1,4 +1,3 @@
-add_subdirectory(linuxdeploy EXCLUDE_FROM_ALL)
 if(NOT TARGET gtest AND BUILD_TESTING)
     add_subdirectory(googletest EXCLUDE_FROM_ALL)
 endif()
Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- CMakeLists.txt	(revision 70ca710c1111a48f03befe184b873022a9c048e2)
+++ CMakeLists.txt	(date 1570482208599)
@@ -3,6 +3,15 @@

 include(CTest)

+if(USE_CONAN)
+    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+    conan_basic_setup(NO_OUTPUT_DIRS)
+endif()
+
+find_package(linuxdeploy REQUIRED)
+find_package(args REQUIRED)
+find_package(Boost REQUIRED modules filesystem)
+
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)

Index: tests/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- tests/CMakeLists.txt	(revision 70ca710c1111a48f03befe184b873022a9c048e2)
+++ tests/CMakeLists.txt	(date 1570482507487)
@@ -13,7 +13,7 @@
 endif()

 add_executable(linuxdeploy-plugin-qt-tests test_main.cpp test_deploy_qml.cpp ../src/qml.cpp)
-target_link_libraries(linuxdeploy-plugin-qt-tests linuxdeploy_core args json gtest linuxdeploy-plugin-qt_util)
+target_link_libraries(linuxdeploy-plugin-qt-tests linuxdeploy_core json gtest linuxdeploy-plugin-qt_util)
 target_compile_definitions(linuxdeploy-plugin-qt-tests PRIVATE TESTS_DATA_DIR="${CMAKE_CURRENT_SOURCE_DIR}/data")

 ld_add_test(linuxdeploy-plugin-qt-tests linuxdeploy-plugin-qt-tests)
Index: src/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/CMakeLists.txt	(revision 70ca710c1111a48f03befe184b873022a9c048e2)
+++ src/CMakeLists.txt	(date 1570482448491)
@@ -1,9 +1,9 @@
 add_library(linuxdeploy-plugin-qt_util OBJECT util.cpp util.h)
 target_include_directories(linuxdeploy-plugin-qt_util PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(linuxdeploy-plugin-qt_util linuxdeploy_core args)
+target_link_libraries(linuxdeploy-plugin-qt_util PUBLIC linuxdeploy::linuxdeploy args::args)

 add_executable(linuxdeploy-plugin-qt main.cpp qt-modules.h qml.cpp qml.h deployment.h)
-target_link_libraries(linuxdeploy-plugin-qt linuxdeploy_core args json linuxdeploy-plugin-qt_util)
+target_link_libraries(linuxdeploy-plugin-qt json linuxdeploy-plugin-qt_util)
 set_target_properties(linuxdeploy-plugin-qt PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/bin")

 add_subdirectory(deployers)
